src/components/LikeButton.jsx

```diff
@@ -0,0 +1,42 @@
+import PropTypes from "prop-types";
+import "./LikeButton.css";
+
+export const LikeButton = ({ thought, setThoughts }) => {
+  const handleLikeClick = () => {
+    const newNum = thought.hearts + 1;
```

Comment:
Could this variable be a bit more descriptive? newNum felt somewhat vague to me personally :)